manager/services/ev_certificate_provider.go

```diff
@@ -19,7 +19,7 @@ type EvCertificateProvider interface {
 	ProvideCertificate(exiRequest string) (EvCertificate15118Response, error)
 }
 
-type HubjectEvCertificateProvider struct {
+type MOEvCertificateProvider struct {
```

Comment:
The details of this implementation are less about MO (that's kind of implied) but about it using the OPCP protocol so would probably call it OPCPEvCertificateProvider.

Reply:
I guess there could be an OpcpOemEvCertificateProvider (although that would be a different interface) - so OpcpMoEvCertificateProvider could be an option.